src/layout/general_settings.rs

```diff
@@ -9,6 +11,15 @@ use serde::{Deserialize, Serialize};
 pub struct GeneralSettings {
     /// The direction which the components are laid out in.
     pub direction: LayoutDirection,
+    /// The font to use for the timer text. `None` means a default font should
+    /// be used.
+    pub timer_font: Option<Font>,
+    /// The font to use for the times and other values. `None` means a default
+    /// font should be used.
```

Comment:
I'm not entirely sure about this:
```suggestion
    /// The font to use for the times and other numerical values. `None` means a
    /// default font should be used.
```

Reply:
It's use for all values of key value components like current comparison. At least at the moment.

Reply:
Alright, we can keep it as is then.